programs/openbook-v2/src/instructions/place_and_cancel_multiple_orders.rs

```diff
@@ -0,0 +1,148 @@
+use anchor_lang::prelude::*;
+use std::cmp;
+
+use crate::accounts_ix::*;
+use crate::accounts_zerocopy::*;
+use crate::error::*;
+use crate::state::*;
+use crate::token_utils::token_transfer;
+
+#[allow(clippy::too_many_arguments)]
+pub fn place_and_cancel_multiple_orders(
```

Comment:
maybe `cancel_and_place_multiple_orders` given that the first action performed is the cancel?